README.md

```diff
@@ -172,8 +172,24 @@ Live inference demos that run on a webcam or video files.
 | -------------------------------------------------------------------------------- | --------------------------------------------------------------------------- | ------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
```

Comment:
Please replace the pose estimation notebook in this highlight table with yours. Remember to keep them in order :)

Reply:
Replaced the new notebook maintaining the order